variable.c

```diff
@@ -1444,7 +1444,8 @@ rb_ivar_lookup(VALUE obj, ID id, VALUE undef)
                     UNLIKELY(!rb_ractor_main_p()) &&
                     !rb_ractor_shareable_p(val)) {
                 rb_raise(rb_eRactorIsolationError,
-                        "can not get unshareable values from instance variables of classes/modules from non-main Ractors");
+                        "can not get unshareable values from instance variables of classes/modules from non-main Ractors (%s from %"PRIsVALUE")",
+                        rb_id2name(id), obj);
```

Comment:
Variable names also have encodings.

```suggestion
                        "can not get unshareable values from instance variables of classes/modules from non-main Ractors (%"PRIsVALUE" from %"PRIsVALUE")",
                        rb_id2str(id), obj);
```